akka-testkit-typed/src/main/scala/akka/testkit/typed/StubbedActorContext.scala

```diff
@@ -81,9 +81,9 @@ private[akka] final class FunctionRef[-T](
    * Do not actually stop the child inbox, only simulate the liveness check.
    * Removal is asynchronous, explicit removeInbox is needed from outside afterwards.
    */
-  override def stop[U](child: ActorRef[U]): Boolean = {
+  override def stop[U](child: ActorRef[U]): Unit = {
```

Comment:
This implementation can be scrapped entirely (and the comment updated) now, right?